full-stack-tests/core/src/frontend/standalone/DisplayStyleState.test.ts

```diff
@@ -59,8 +64,17 @@ describe("DisplayStyle", () => {
     expect(style2.sunDirection).not.to.be.undefined;
     expect(style2.sunDirection.isAlmostEqual(style1.sunDirection)).to.be.true;
 
+    userToken = "bad guy";
```

Comment:
What does any of this have to do with preserving the sun direction? If you want to test tokens, make a test that specifically tests tokens.

Reply:
done

Reply:
@kabentley Thank you!